.eslintrc.cjs

```diff
@@ -76,6 +76,7 @@ module.exports = {
       'react/react-in-jsx-scope': 'off',
       // additional rules:
       '@typescript-eslint/no-floating-promises': 'error',
+      "import/prefer-default-export": "off",
```

Comment:
Please verify if any of the changed files can be removed from eslintignore

Reply:
have only added new files.